processor/src/main/java/org/mapstruct/ap/internal/model/PropertyMapping.java

```diff
@@ -203,6 +203,7 @@ protected boolean isFieldAssignment() {
         // initial properties
         private String defaultValue;
         private SourceReference sourceReference;
+        private SourceRHS rhs;
```

Comment:
Yes I saw that somewhere we use `rightHandSide` and now also `rhs`. I noticed this because some method parameters where you pass the `rhs` were renamed from `rightHandSide` to `rhs` 😄. Maybe use everywhere `rhs` or `rightHandSide` and then we are consistent